### ISSUE TITLE
create-tar.sh should also fetch the timestamp

### ISSUE THREAD

**Issue by msirringhaus** [openSUSE/firefox-scripts#7]
`create-tar.sh` already parses which build-candidate is the release (which is done stupidly, but I know of no direct way to do this), so it could also directly parse the timestamp for the spec-file from:
http://ftp.mozilla.org/pub/firefox/candidates/68.2.0esr-candidates/build1/linux64_info.txt

**Comment by msirringhaus** [comment]
Actually, even better would be:

http://ftp.mozilla.org/pub/firefox/candidates/68.2.0esr-candidates/build1/linux-x86_64/en-US/firefox-68.2.0esr.txt

Which contains the build-id and the build-revision.

**Comment by msirringhaus** [comment]
This is now in the "dev"-branch. Once current submissions are released, it will be merged to master.

**Comment by msirringhaus** [comment]
Once this is done, we can also do https://github.com/openSUSE/firefox-maintenance/issues/16

as source-stamp.txt is no longer needed (tar_stamps will contain all the info)
